tests/expected/function-contract/attribute_no_complain.expected

```diff
@@ -0,0 +1 @@
+VERIFICATION:- SUCCESSFUL
```

Comment:
Maybe you should combined these little examples that should succeed into a single file. Also, fix the missing lines.

Reply:
I'm not sure that's a good idea. I don't think we have a good way to distinguish to which example each output line corresponds to if we combine them and I don't really see the benefit in combining it either.

Reply:
We could track successful property from different harness using the Kani output. For instance,
```
Check 2: dig_deeper.assertion.1
         - Status: SUCCESS
         - Description: "assertion failed: ptr(input) as usize == 10"
         - Location: foreign_functions.rs:34:9 in function dig_deeper
```
Here, we have location and property. That should be enough to differ them using regex in the `*.expected` files. Am I missing something? In any case, we don't need to block this PR on this. It's just a clean up.

Reply:
`*.expected` files don't support regex. 

Also @celinval specifically requested not to use locations in `*.expected` files. They used to be in there and she requested I remove them which I think was sensible because at least the file path would come out slightly differently depending on how you called the test binary.

Similarly relying on the check number or property type is, in my opinion, brittle.